python/katana/local/entity_type_manager.pyx

```diff
@@ -0,0 +1,13 @@
+cdef class EntityTypeManager:
+
+    @staticmethod
+    cdef EntityTypeManager make(const CEntityTypeManager *manager):
+        m = <EntityTypeManager>EntityTypeManager.__new__(EntityTypeManager)
+        m.underlying_entity_type_manager = manager
+        return m
+
+    def get_atomic_entity_type_ids(self):
+        """
+        Return a list of all entity type IDs
+        """
+        return self.underlying_entity_type_manager.GetAtomicEntityTypeIDs()
```

Comment:
Since we are exposing this to the user, please provide a complete API. This doesn't mean you need to wrap every method. But this wrapper should provide access to all the features of the underlying class.